### ISSUE TITLE
Training model on Interhand-3d is too slow

### ISSUE THREAD

**Issue by Whuhao** [open-mmlab/mmpose#1019]
**Describe the issue**

I follow the instruction and sucessfully run the code on my computer, but the training speed is slow. When I tried to train a model that has resnet18/34/50 backbone, it was estimated that it would take 30 days to finish. 

I wonder the reason why it would take so long?
And how long will it take for you to train the same modle?

Below is a example of Resnet18, using a single gpu. However, mutil-gpu does not make it much better.

**Environment**

sys.platform: linux
Python: 3.7.10 (default, Jun  4 2021, 14:48:32) [GCC 7.5.0]
CUDA available: True
GPU 0: Tesla M40 24GB
CUDA_HOME: /ssd1/shared/local/cuda-10.1
NVCC: Cuda compilation tools, release 10.1, V10.1.105
GCC: gcc (GCC) 4.8.5 20150623 (Red Hat 4.8.5-36)
PyTorch: 1.9.0
PyTorch compiling details: PyTorch built with:
  - GCC 7.3
  - C++ Version: 201402
  - Intel(R) oneAPI Math Kernel Library Version 2021.2-Product Build 20210312 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v2.1.2 (Git Hash 98be7e8afa711dc9b66c8ff3504129cb82013cdb)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - NNPACK is enabled
  - CPU capability usage: AVX2
  - CUDA Runtime 11.1
  - NVCC architecture flags: -gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_61,code=sm_61;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_80,code=sm_80;-gencode;arch=compute_86,code=sm_86;-gencode;arch=compute_37,code=compute_37
  - CuDNN 8.0.5
  - Magma 2.5.2
  - Build settings: BLAS_INFO=mkl, BUILD_TYPE=Release, CUDA_VERSION=11.1, CUDNN_VERSION=8.0.5, CXX_COMPILER=/opt/rh/devtoolset-7/root/usr/bin/c++, CXX_FLAGS= -Wno-deprecated -fvisibility-inlines-hidden -DUSE_PTHREADPOOL -fopenmp -DNDEBUG -DUSE_KINETO -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DSYMBOLICATE_MOBILE_DEBUG_HANDLE -O2 -fPIC -Wno-narrowing -Wall -Wextra -Werror=return-type -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-sign-compare -Wno-unused-parameter -Wno-unused-variable -Wno-unused-function -Wno-unused-result -Wno-unused-local-typedefs -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-psabi -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Wno-stringop-overflow, LAPACK_INFO=mkl, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, TORCH_VERSION=1.9.0, USE_CUDA=ON, USE_CUDNN=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=ON, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON,

TorchVision: 0.2.2
OpenCV: 4.5.4-dev
MMCV: 1.3.17
MMCV Compiler: GCC 7.3
MMCV CUDA Compiler: 11.1
MMPose: 0.20.0+c0d5e4c

**Results**

2021-11-16 11:53:53,570 - mmpose - INFO - Epoch [1][380/85067]  lr: 2.000e-04, eta: 29 days, 23:56:45, time: 1.553, data_time: 1.300, memory: 3810, hand_loss: 11.1313, rel_root_loss: 1.4328, hand_type_loss: 0.4370, acc_classification: 0.5405, loss: 13.0011
2021-11-16 11:54:24,253 - mmpose - INFO - Epoch [1][400/85067]  lr: 2.000e-04, eta: 30 days, 0:11:04, time: 1.534, data_time: 1.280, memory: 3810, hand_loss: 11.1102, rel_root_loss: 0.8791, hand_type_loss: 0.4050, acc_classification: 0.5426, loss: 12.3942
2021-11-16 11:54:54,691 - mmpose - INFO - Epoch [1][420/85067]  lr: 2.000e-04, eta: 30 days, 0:08:17, time: 1.523, data_time: 1.269, memory: 3810, hand_loss: 11.5023, rel_root_loss: 1.0939, hand_type_loss: 0.3828, acc_classification: 0.5229, loss: 12.9790

**Comment by Whuhao** [comment]
I may have figured the problem out, it is because the config set the defaut cpu_worker as 1. As a result, we have to spend a lot of time waiting for it.
